vscode/microsoft-kiota/src/openApiTreeProvider.ts

```diff
@@ -350,6 +350,7 @@ export class OpenApiTreeProvider implements vscode.TreeDataProvider<OpenApiTreeN
                     }
                     await updateTreeViewIcons(treeViewId, true, false);
                 }
+                void vscode.window.showInformationMessage(vscode.l10n.t('You can now select the required endpoints from ') + this.apiTitle);
```

Comment:
this should be using a string format instead of concat since in different languages, the name of the service might not always be at the end of the sentence.

Reply:
How would that look like?

Reply:
Here is an example https://github.com/microsoft/kiota/blob/b0d06e39f399c68e59b672995a2992def1099f25/vscode/microsoft-kiota/src/extension.ts#L703